ext/adaptors/ranges-lib.tex

```diff
@@ -239,17 +246,20 @@
 
 \begin{itemdescr}
 \pnum
-\effects Equivalent to \tcode{return ranges::begin(derived()) == ranges::end(derived());}.
+\effects Equivalent to: \tcode{return ranges::begin(derived()) == ranges::end(derived());}.
```

Comment:
Perhaps this should be `ranges::empty(derived())`.

Reply:
That would infinitely recurse if `derived()` does not override `empty`.